src/ui/actions/timeline.ts

```diff
@@ -337,7 +341,7 @@ function playback(startTime: number, endTime: number): UIThunkAction {
     const shouldContinuePlayback = () => getPlayback(getState());
     prepareNextGraphics();
 
-    Video.play();
+    Video.play(currentTime);
```

Comment:
Is this value the same as what `protocol/graphics` was using before?

https://github.com/RecordReplay/devtools/blob/ae52cb0d709e2a1b480697d8afa9243b05c60f8a/packages/protocol/graphics.ts#L232

Reply:
It looks like it could be different. It'd be easier to say if there were a replay :)

Reply:
Okay for now I just matched the pre-existing behavior for this callsite.